app/models/user.rb

```diff
@@ -0,0 +1,9 @@
+class User < ApplicationRecord
+  has_many :posts, foreign_key: :author_id
```

Comment:
You have done great so far :+1: :+1: But there are a few changes you need to make before moving to the next milestone. :+1: :100: 

- [ ] THe relationship you defined between user and post here is correct but the syntax is wrong, the `author id` is meant to be in a string. here's how it should look like.
```ruby
has_many :posts, foreign_key: 'author_id'
```